### PR TITLE
[OS-911] Adding Jenkinsfile for DR automatic builds

### PR DESCRIPTION
**Pull request by skarbat**

### CODE REVIEW

**Review by radujipa — APPROVED** [review]
:shipit:

File: bin/kano-jupyter [hunk 1]
```diff
@@ -87,6 +87,7 @@ if __name__ == '__main__':
     args = docopt.docopt(__doc__)
 
     if args['run']:
+        os.system('kdesk-hourglass-app "Chromium"')
```
Comment (tombettany) [inline bin/kano-jupyter:90]: Chromium?

**Review by tombettany — APPROVED** [review]
Nice

File: bin/kano-jupyter (hunk 1 above)
Comment (skarbat) [reply bin/kano-jupyter:90]: It means start the hourglass until a window comes up with the word "Chromium" somewhere on it's titlebar.

Comment (tombettany) [reply bin/kano-jupyter:90]: I didn't realise that the Jupyter stuff opened in Chromium but actually it makes complete sense.

Cool, no worries.

**Pull request merged**